### ISSUE TITLE
Analyse Fable tests and log issues

### ISSUE THREAD

**Issue by connorscarolyns** [cds-snc/notification-planning#2071]

**Comment by connorscarolyns** [comment]
@schancase I have reviewed the tests and made a spreadsheet of the issues identified. I think maybe the next step should be to review the issues with the team, maybe as a watch party? and decide which of the issues should be logged.

**Comment by connorscarolyns** [comment]
Because these are articles pages, maybe the watch party is not needed, I could review with fill and create issues for a simpler path forward.

**Comment by connorscarolyns** [comment]
Complete, tickets have been logged and created
